ethicml/data/dataset.py

```diff
@@ -152,15 +161,17 @@ def load(self, ordered: bool = False) -> DataTuple:
         assert isinstance(dataframe, pd.DataFrame)
 
         feature_split = self.feature_split if not ordered else self.ordered_features
-        feature_split_x = feature_split["x"]
+        if keep_labels_as_features:
+            feature_split_x = feature_split["x"] + feature_split["s"] + feature_split["y"]
+        else:
+            feature_split_x = feature_split["x"]
```

Comment:
(the above is the main change)

Reply:
should there be the option to just have S, but not Y, included in the features?